driver/pelican/params.yml

```diff
@@ -4,3 +4,4 @@ password: <pelican password>
 sitename: <pelican site name>
 name: <thermostat name>
 poll_interval: <status poll interval>
+timezone: <IANA timezone> # Must Be Recognized IANA Timezone Format
```

Comment:
Hmm, I think we can leave the comment out here actually.

The other fields can have an expected syntax as well, e.g. `poll_interval` needs to be a valid Go-style time duration string, but we don't have comments for them, so let's just be consistent.

Reply:
Nvm, sounds like we want to leave this in.